src/bin/qasphere.ts

```diff
@@ -2,5 +2,16 @@
 
 import { hideBin } from 'yargs/helpers'
 import { run } from '../commands/main'
+import { validateNodeVersion } from '../utils/misc'
+import { REQUIRED_NODE_VERSION } from '../utils/config'
+import chalk from 'chalk'
+
+if (!validateNodeVersion()) {
```

Comment:
Checks for a minimum required nodejs version. Prints a warning if the node version used is older than the required version.